{"name": "Проверка оформления заказа без id заказа", "status": "failed", "statusDetails": {"message": "AssertionError: assert 404 == 400\n +  where 404 = <Response [404]>.status_code", "trace": "self = <tests.test_accept_order.TestGetOrder object at 0x000001AA707D2790>, register_new_courier_and_return_login_password = [['ivgzhagkjt', 'xxfptctlpm', 'jvkoqkddmo'], <Response [201]>]\n\n    @allure.title('Проверка оформления заказа без id заказа')\n    def test_accept_order_without_id_order(self, register_new_courier_and_return_login_password):\n    \n        payload = {\n            \"login\": register_new_courier_and_return_login_password[0][0],\n            \"password\": register_new_courier_and_return_login_password[0][1]\n        }\n        # получаем id курьера\n        api = ApiClient()\n        response_post = api.post(path=ApiPath.path_id_courier_login, payload=payload)\n        id_courier = response_post.json()['id']\n        get_params_id_courier = {\"courierId\": id_courier}\n        # принимаем заказ со всеми полученными параметрами\n        response_put = api.put(path=ApiPath.path_accept_order, get_params=get_params_id_courier)\n    \n>       assert response_put.status_code == 400\nE       assert 404 == 400\nE        +  where 404 = <Response [404]>.status_code\n\ntests\\test_accept_order.py:64: AssertionError"}, "start": 1703513983011, "stop": 1703513983237, "uuid": "2266290f-03ca-4681-976d-71dd21c6a29c", "historyId": "8f15678ad05029a1e459df7528315815", "testCaseId": "8f15678ad05029a1e459df7528315815", "fullName": "tests.test_accept_order.TestGetOrder#test_accept_order_without_id_order", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_accept_order"}, {"name": "subSuite", "value": "TestGetOrder"}, {"name": "host", "value": "inetshop-61"}, {"name": "thread", "value": "16164-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_accept_order"}]}